src/dfi/govvariables/attributes.cpp

```diff
@@ -817,6 +837,9 @@ const std::map<uint8_t, std::map<uint8_t, std::function<ResVal<CAttributeValue>(
                  {DFIPKeys::TransferDomain, VerifyBool},
                  {DFIPKeys::LiquidityCalcSamplingPeriod, VerifyMoreThenZeroInt64},
                  {DFIPKeys::AverageLiquidityPercentage, VerifyPctInt64},
+                 {DFIPKeys::EmissionReduction, VerifyMoreThenZeroUInt32},
+                 {DFIPKeys::TargetSpacing, VerifyMoreThenZeroInt64},
+                 {DFIPKeys::TargetTimespam, VerifyMoreThenZeroInt64},
```

Comment:
On Purpose VerifyInt and not Uint ? Does Int64 for a positive number even make sense?

Reply:
We are retaining the same type, nTargetTimespan and nTargetSpacing are int64_t.